platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/PkiTools.java

```diff
@@ -266,6 +290,8 @@ public static X509Certificate pemToCertificate(String certString) {
      */
     public static byte[] pemToDer(String string) {
         Validate.isTrue(string != null, "PEM string cannot be null");
+        assert string != null;
```

Comment:
It lets the IDE know to shut up. It's suboptimal, but an effective noop so I continued the pattern of using it that has already been established in this code.